content/live-disk.md

```diff
@@ -98,27 +98,37 @@ Press the <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon>
 
 ### For Windows/macOS
 
-Etcher is an open source app for Windows and macOS that allows you to "burn images to SD cards & USB drives safely and easily." You can download it at [Etcher.io](https://etcher.io).
+Etcher is an open source app for Windows, macOS, and Linux that allows you to burn disk images to USB drives. You can download it at [balena.io/etcher/](https://www.balena.io/etcher/).
 
 Once you have installed Etcher and downloaded the Pop!_OS.iso image, open up the Etcher application. You should see something like this:
 
-![Etcher](/images/live-disk/etcher.png)
+![Etcher Startup](/images/live-disk-new/etcher01-start.png)
 
-Choose `Select Image` and navigate to where you downloaded Pop!_OS, click on it and hit the `Open` button:
+Choose `Flash from file` and then navigate to where you downloaded the Pop!_OS ISO file (typically your "Downloads" folder). Click on the file then and hit the `Open` button. On Windows, it should look something like this:
 
-![Select Pop!_OS iso](/images/live-disk/open-pop-iso-etcher.png)
+![Select Pop!_OS iso](/images/live-disk-new/etcher02-filepicker.png)
 
-Next, select the drive that you want to use. If there is only one, it should automatically recognize that you have a flash drive inserted and select it.
+Now click on `Select Target`:
 
-![Etcher Flash Drive Selected](/images/live-disk/etcher-flash-selected.png)
+![Click on Select Target](images/live-disk-new/etcher03-select-target.png)
 
-Now hit the `Flash` button and watch the magic happen!
+Next, select the drive that you want to use by checking the box next to it. It should only show removeable drives by default. After the box for your USB stick is checked, click on the `Select` button:
```

Comment:
typo: removable

Reply:
I always get that word wrong, and I meant to spellcheck it before I did the PR. Fixed.